### PR TITLE
Provide test templates for Hibernate Search (Lucene and Elasticsearch)

### PR DESCRIPTION
**Pull request by gsmet**
Here are 2 test templates for Hibernate Search, one for Lucene, one for Elasticsearch.

### CODE REVIEW

File: search/hibernate-search-lucene/pom.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.hibernate.testcasetemplate</groupId>
+	<artifactId>test-case-template-hibernate-search-lucene</artifactId>
+	<version>1.0.0.Final</version>
+	<name>Hibernate Search with Lucene Test Case Template</name>
+
+	<properties>
+		<version.org.hibernate.search>5.6.0.Beta1</version.org.hibernate.search>
```
Comment (gunnarmorling) [inline search/hibernate-search-lucene/pom.xml]: Latest stable here maybe?

**Comment by gunnarmorling** [comment]
Looks good, go ahead and merge from my POV. If you see any potential for slimming down (e.g. FEST assert?!), I'd be all for it. The smaller, the easier to grasp and the better.

Could you create the same for OGM and HV, too? I think there are JIRA issues existing already. Thanks a lot!

**Comment by gsmet** [comment]
@gunnarmorling the idea is to be able to include the tests directly so FEST assert looked like a good idea. But maybe I should stick to basic JUnit?

File: search/hibernate-search-lucene/pom.xml (hunk 1 above)
Comment (gsmet) [inline search/hibernate-search-lucene/pom.xml]: I wanted to propose a test case for Elasticsearch too and I thought they might as well be consistent.

But I can use the last stable for Lucene if your prefer. I don't have a strong opinion about it.

Comment (gunnarmorling) [inline search/hibernate-search-lucene/pom.xml]: Yes, I'd stick to plain JUnit in this cae.

Comment (gunnarmorling) [inline search/hibernate-search-lucene/pom.xml]: The stable will be what most people use (besides adventurous ES explorers),
so I'd stick to that for Lucene.

**Comment by gsmet** [comment]
OK, moved the Lucene template to 5.5.3.Final and removed the fest assert dependency.

**Pull request merged**